provider/resource_kpi_threshold_template.go

```diff
@@ -73,7 +73,7 @@ type modelKpiThresholdTemplate struct {
 }
 
 type TimeVariateThresholdsSpecificationModel struct {
-	Policies []PolicyModel `tfsdk:"policies"`
+	Policies types.Set `tfsdk:"policies"`
```

Comment:
it probably requires the same for all sets:
- Policies
- TimeBlocks
- KpiThresholdLevelModel

Reply:
done